### ISSUE TITLE
v

### ISSUE THREAD

**Issue by jimblue** [bhilburn/powerlevel9k#1047]
Hi,

Since I've update from `v0.6.4` to latest on master branch os icon does work anymore.

![screenshot 2018-11-02 at 04 52 21](https://user-images.githubusercontent.com/10623367/47893093-7cae7b80-de5b-11e8-9fd9-a5fdf5c61948.png)

After some test the bug appear since `v0.6.5` and remain on `v0.6.6`.

Thanks for you help!

**Comment by bhilburn** [comment]
@jimblue - Are you using Nerdfonts, by chance?

Nerdfonts updated a lot of codepoints in a major version release, and we bumped the default version in v0.6.5, I think.

**Comment by jimblue** [comment]
Didn't think about that update the font...
Thank it solve problem!

Also with v0.6.6, the context background color setup to be white is now black...
Is there a new context input I missed? How can I get the white background back?

**Comment by jimblue** [comment]
As it's not working anymore and I can't found the following option in the doc, are they still available

**Comment by dritter** [comment]
Yep, they are still available. But there are more states now. Have a look at https://github.com/bhilburn/powerlevel9k/issues/898#issuecomment-403967072 where I explain the new ones.

**Comment by bhilburn** [comment]
@jimblue - It's still there in the docs, unless you're looking at something different?
https://github.com/bhilburn/powerlevel9k#context

I'm going to close out this issue since your original problem has been solved. If your colors aren't working now, can you open a new bug report?

**Comment by jimblue** [comment]
@bhilburn I've already close this! Everything is working well now!
Thanks for your help

**Comment by bhilburn** [comment]
@jimblue - Great! Happy to hear it =)
